docs/how-to/use-rewards-api.md

```diff
@@ -32,7 +32,7 @@ The following limitations apply:
 
 - The rewards API relies on state and block data to retrieve the reward information, meaning you'll receive limited data if the beacon node being queried is not an archive node.
 
-- You can only query blocks from finalized to head if you are in `prune` storage mode.
```

Comment:
We shouldn't change this.

Reply:
this would be a candidate for 'if you are not using archive storage mode' rather than saying 'prune storage mode'